parsl/executors/high_throughput/executor.py

```diff
@@ -626,7 +626,6 @@ def outstanding(self) -> int:
         and managers"""
         return len(self.tasks)
 
-    @property
     def connected_workers(self) -> int:
```

Comment:
Thank you!  I wasn't sure how to handle this case, and had it on a local stack to get back to.  In particular, I was observing that even the `hasattr(..., "connected_workers")` was invoking the logic, which is not, I think, what the call-site expects.